### DISCUSSION TITLE
Support for go modules

### DISCUSSION THREAD

**Discussion by mtestrot** [aquasecurity/trivy#938] [Q&A] [answered]
Hi,

I just stumbled across this nice project and wondered why go modules are not supported when scanning application dependencies https://aquasecurity.github.io/trivy/latest/vuln-detection/library/

After all trivy is written in Go.

Perhaps there is a blocker I don't see or there is already an established alternative for Go?
I'm just curious.

Thx for any answer.

**Comment by tomadele** [comment]
Is there any news about this ?

**Comment by knqyf263** [comment, marked as answer]
Please watch https://github.com/aquasecurity/trivy/issues/958

**Reply by knqyf263** [reply]
We released v0.17.0 supporting Go binaries.

**Reply by mtestrot** [reply]
Awesome. Thx!
